src/core/scene/a-scene.js

```diff
@@ -312,10 +397,36 @@ module.exports = registerElement('a-scene', {
 
         if (this.isPlaying) {
           this.tick(time, timeDelta);
+          this.tock(time, timeDelta);
+        }
+
+        if (this.behaviors.tickPostRender.length || this.behaviors.tockPostRender.length) {
+          // Post processing setup has been modified, emit a notification.
+          if (this.postNeedsNotify) {
+            var behaviors = this.behaviors;
+            this.postNeedsNotify = false;
+            this.emit('post-modified', {
+              tick: behaviors.tickPostRender,
+              tock: behaviors.tockPostRender
+            });
+          }
+
+          this.effect.render(this.object3D, camera, this.renderTarget);
+
+          this.time = time;
+          time = window.performance.now();
+          timeDelta = time - this.time;
```

Comment:
I think we can just reuse the timeDelta from before. Makes sense that it's timeDelta between each frame